transcom/mymove
# INT B-19886 Hide Diversion Link for PPMs

## [B-19886](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19866)

## Summary

Does what it says on the tin.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

General Testing Instructions: 
1. As any office user access any move containing a PPM.
2. Navigate to the MTO page.
3. Ensure the Authorized Addresses card for the PPM does not contain the "Request diversion" button.
4. Find any move with an approved HHG
5. Ensure the HHG's Authorized Addresses card does still contain the "Request diversion" button.

Specific Testing Instructions
1. Access the office app
2. Login as a TOO
3. Search for the move #CLOSE1, which contains a PPM
4. Open that move and navigate to the Move Task Order page
5. Verify the Authorized Addresses card for the PPM **does not** contain the "Request diversion" button.
6. Repeat with move #SITFUT, which contains an HHG, ensuring it **does** contain the "Request diversion" button.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
